code/controllers/subsystem/traumas.dm

```diff
@@ -13,7 +13,7 @@ SUBSYSTEM_DEF(traumas)
 /datum/controller/subsystem/traumas/Initialize()
 	phobia_types = list("spiders", "space", "security", "clowns", "greytide", "lizards",
 						"skeletons", "snakes", "robots", "doctors", "authority", "the supernatural",
-						"aliens", "strangers")
+						"aliens", "strangers", "birds", "falling")	//doesn't contain conspiracies?
```

Comment:
Put the comment there to remind me. Conspiracies is in the list of phobia words below but not in the phobia types. Why?

Reply:
Conspiracies is a special trauma you can only obtain via the tinfoil hat. Having it in the list means it can be pulled for brain damage, which I didn't want.

Reply:
Alright I'll remove my comment. Ta for saying!